common/views/components/CardGrid/CardGrid.js

```diff
@@ -15,6 +15,7 @@ import { type Book } from '../../../model/books';
 import { type Article } from '../../../model/articles';
 import Space from '../styled/Space';
 import type Moment from 'moment';
+import React from 'react';
```

Comment:
Not that it matters hugely but we usually just import { Fragment } from 'react';